### PR TITLE
Support only current python versions.

### PR DESCRIPTION
**Pull request by sallner**
Update travis and tox setup to only support the currently officially supported Python versions. 

Allthough not much is changing, it might be a good idea to cut a release after the merge to show the compatibility on pypi.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,11 +3,11 @@ language: python
...
 
 env:
-  - TOXENV=py26
   - TOXENV=py27
-  - TOXENV=py32
-  - TOXENV=py33
   - TOXENV=py34
+  - TOXENV=py35
+  - TOXENV=py36
+  - TOXENV=py37
```
Comment (tseaver) [inline .travis.yml]: Does TravisCI support Python 3.7 out-of-the-box?

Comment (stevepiercy) [reply .travis.yml]: Sort of. There are workarounds as indicated in [this thread](https://github.com/travis-ci/travis-ci/issues/9815). [Pyramid is in a holding pattern](https://github.com/Pylons/pyramid/issues/3312#issuecomment-403138089).

**Comment by stevepiercy** [comment]
I created a [checklist of files to review when adding/dropping Python version support in Pyramid](https://github.com/Pylons/pyramid/issues/3312#issue-338744276) that may be useful for this PR, too.

The `README.rst` needs a bump, and while you're in there, the link to docs needs to use `https`. Same link needs to use `https` in [`setup.py`](https://github.com/Pylons/peppercorn/blob/master/setup.py#L55)

Also please sign `CONTRIBUTORS.txt`.

**Comment by stevepiercy** [comment]
['index.rst`](https://github.com/Pylons/peppercorn/blob/master/docs/index.rst) needs a bump, too.

**Comment by tseaver** [comment]
We need to tweak `.travis.yml` to get the necessary Python interpreters added, e.g. something like [Webob's `.travis.yml`](https://github.com/Pylons/webob/blob/master/.travis.yml).

File: .travis.yml (hunk 1 above)
Comment (sallner) [reply .travis.yml]: I adapted the `travis.yml` from Webob.

**Comment by sallner** [comment]
I think I did all the requested changes and signed the `CONTRIBUTORS.txt`. So this should be fine now. Thanks for the review.

File: .travis.yml [hunk 2]
```diff
@@ -2,15 +2,24 @@
...
+        - python: 2.7
+          env: TOXENV=py27
+        - python: 3.4
+          env: TOXENV=py34
+        - python: 3.5
+          env: TOXENV=py35
+        - python: 3.6
+          env: TOXENV=py36
+        - python: nightly
+          env: TOXENV=py37
```
Comment (tseaver) [inline .travis.yml]: I believe that `nightly` should actually be a 3.8 snapshot.  Can you look at what e.g. [`zope.interface` is doing to support 3.7](https://github.com/zopefoundation/zope.interface/blob/e6693826ab7e8c3b640763f65da2f28fe89a961a/.travis.yml#L13).  I realize that WebOb isn't yet doing that.

**Comment by sallner** [comment]
@tseaver So, finally working with Python 3.7.

**Comment by tseaver** [comment]
@sallner Thank you for the patch!

**Comment by sallner** [comment]
You're welcome. I would be very glad, if you could cut a release, maybe after #14 is merged, as would help with the Python 3 migration of a project.

**Comment by icemac** [comment]
@mcdonc @tseaver @stevepiercy As now all issues and PRs in this package are closed/resolved, can we have a new release? (You three currently have the needed PyPI rights.) Thanks in advance.

**Comment by stevepiercy** [comment]
It's about time I started doing a few more releases of Pylons Project projects to PyPI.

@tseaver can you review this checklist for me to make sure I don't overlook anything? If it looks good, I'll cut a release today.

- [ ] Do platform test via tox: `tox -r`
- [ ] `CHANGES.rst` is updated with release date
- [ ] Change `setup.py` version to the release version number. (0.6)
- [ ] Make sure PyPI long description renders (requires ``readme_renderer`` installed into your Python): `python setup.py check -r -s -m`
- [ ] Create a release tag.
- [ ] Make sure your Python has ``setuptools-git``, ``twine``, and ``wheel`` installed and release to PyPI:

  ```bash
  python setup.py sdist bdist_wheel
  twine upload dist/peppercorn-0.6
  ```
- [ ]  Announce to Twitter and pylons-discuss

**Comment by stevepiercy** [comment]
> Consider the release number (bugfix-only vs. features, changes in platforms).

I use [Pyramid's RELEASING.txt](https://github.com/Pylons/pyramid/blob/master/RELEASING.txt#L1-L15) for guidance. I pulled my checklist from that.

Since we're adding/dropping platform support, I reckon 0.6 is the proper number. If not, let me know, and I'll change my branch/PR.

I'll merge your suggestions and publish a RELEASING.txt, too. Thank you!

**Comment by stevepiercy** [comment]
@icemac released!
https://pypi.org/project/peppercorn/

**Comment by icemac** [comment]
@stevepiercy Thank you very much!

**Pull request merged**